include/graylog_logger/LogUtil.hpp

```diff
@@ -108,24 +108,32 @@ class BaseLogHandler {
 public:
   /// \brief Does minimal set-up of the this class.
   /// \param[in] MaxQueueLength The maximum number of log messages stored.
-  explicit BaseLogHandler(const size_t MaxQueueLength = 100);
+  BaseLogHandler() = default;
   virtual ~BaseLogHandler() = default;
+
   /// \brief Called by the logging library when a new log message is created.
-  /// \note If the queue of messages is full, any new messages are discarded
-  /// without any indication that this has been done.
+  ///
+  /// Must be implement by any derived classes.
```

Comment:
```suggestion
```
Enforced by the compiler, so unnecessary?

Reply:
Right.
```suggestion
```